csharp/Adapter/Microsoft.Spark.CSharp/Streaming/PairDStreamFunctions.cs

```diff
@@ -678,6 +683,18 @@ internal RDD<dynamic> Execute(double t, RDD<dynamic> stateRDD, RDD<dynamic> valu
 
             if (stateRDD == null)
             {
+                if (initialState != null)
```

Comment:
@skaarthik CodeCov report shows that this block has been covered by UT.
https://codecov.io/gh/Microsoft/Mobius/compare/4f39658e4e68bb59c7960462ba5724895629101f...1cde355f3bd03634703f78e0f27ea9a32f71ab50#6373686172702F416461707465722F4D6963726F736F66742E537061726B2E4353686172702F53747265616D696E672F506169724453747265616D46756E6374696F6E732E6373-705